Face expressions CMakeLists not installing the configuration xml

### Bug description

After making some tests on `iCubGenova11` with @martinaxgloria, we have doubts on the fact that the `CMakeLists.txt` within `app/faceExpressions`

https://github.com/robotology/icub-main/blob/18b7803391a871e5ad173e8a5546d17d73814ec8/app/faceExpressions/CMakeLists.txt#L6

is only installing configuration files having `ini` extension and not `xml`, such as https://github.com/robotology/icub-main/blob/master/app/faceExpressions/conf/serialPortFaceExpressions.xml.

We tried changing the `CMakeLists.txt` to also include the xml and we noticed that, in such case - and if that action is correct - the file would be installed within the `faceExpressions` context. For that reason, we also think that the following template xml file

https://github.com/robotology/icub-main/blob/18b7803391a871e5ad173e8a5546d17d73814ec8/app/faceExpressions/scripts/faceExpressions.xml.template#L5

is missing a `--context faceExpressions` to make everything works out of the box.



### Steps to reproduce

Considering the current state of the files, when one tries to run 

```console
yarprobotinterface --config serialPortFaceExpressions.xml
```

the following errors are printed:

```console
[DEBUG] Reading file 
[ERROR] Syntax error while loading  at line 0 . Failed to open file
[FATAL] Config file    not parsed correctly.
Trace requested at /usr/local/src/robot/robotology-superbuild/src/YARP/src/yarprobotinterface/Module.cpp:132 by code called from:
/usr/local/src/robot/robotology-superbuild/build/install/lib/libYARP_os.so.3(_Z16yarp_print_traceP8_IO_FILEPKcj+0x3b) [0x7f37d0596b0b]
yarprobotinterface(+0x9038) [0x55a30ed8d038]
yarprobotinterface(+0xa439) [0x55a30ed8e439]
/usr/local/src/robot/robotology-superbuild/build/install/lib/libYARP_os.so.3(_ZN4yarp2os8RFModule9runModuleERNS0_14ResourceFinderE+0x77) [0x7f37d05f2207]
yarprobotinterface(+0x8b90) [0x55a30ed8cb90]
/lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf3) [0x7f37d01060b3]
yarprobotinterface(+0x8c5e) [0x55a30ed8cc5e]
```

as the file cannot be found.

### Expected behavior

_No response_

### Example repository

_No response_

### Additional context

_No response_

--- a/app/faceExpressions/CMakeLists.txt
+++ b/app/faceExpressions/CMakeLists.txt
@@ -3,9 +3,9 @@ set(appname faceExpressions)
 file(GLOB scripts ${CMAKE_CURRENT_SOURCE_DIR}/scripts/*.template
                   ${CMAKE_CURRENT_SOURCE_DIR}/scripts/*.xml
                   ${CMAKE_CURRENT_SOURCE_DIR}/scripts/*.sh)
-file(GLOB conf ${CMAKE_CURRENT_SOURCE_DIR}/conf/*.ini)
+file(GLOB conf ${CMAKE_CURRENT_SOURCE_DIR}/conf/*.ini
+               ${CMAKE_CURRENT_SOURCE_DIR}/conf/*.xml)
                            
 yarp_install(FILES ${conf} DESTINATION ${ICUB_CONTEXTS_INSTALL_DIR}/${appname})
 yarp_install(FILES ${scripts} DESTINATION ${ICUB_APPLICATIONS_TEMPLATES_INSTALL_DIR})
-
 

--- a/app/faceExpressions/scripts/faceExpressions.xml.template
+++ b/app/faceExpressions/scripts/faceExpressions.xml.template
@@ -2,7 +2,7 @@
 <name>Face Expressions</name>
 <module>
       <name>yarprobotinterface</name>
-      <parameters>--config serialPortFaceExpressions.xml</parameters>
+      <parameters>--context faceExpressions --config serialPortFaceExpressions.xml</parameters>
 	  <node>pc104</node>
 	<tag>face_device</tag>
    </module>

